internal/datacoord/index_service.go

```diff
@@ -235,21 +235,25 @@ func (s *Server) CreateIndex(ctx context.Context, req *indexpb.CreateIndexReques
 	// exclude the mmap.enable param, because it will be conflicted with the index's mmap.enable param
 	typeParams := DeleteParams(req.GetTypeParams(), []string{common.MmapEnabledKey})
 	index := &model.Index{
-		CollectionID:    req.GetCollectionID(),
-		FieldID:         req.GetFieldID(),
-		IndexID:         indexID,
-		IndexName:       req.GetIndexName(),
-		TypeParams:      typeParams,
-		IndexParams:     req.GetIndexParams(),
-		CreateTime:      req.GetTimestamp(),
```

Comment:
we use current schema version as the minSchemaVersion for the index
as the schema on the dataCoord is always up-to-date, so this version decision is robust